.travis.yml

```diff
@@ -11,7 +11,7 @@ install:
   - mvn install -DskipTests=true -Dmaven.javadoc.skip=true -B -V
 
 script:
-  - mvn clean test cobertura:cobertura coveralls:report -B -V
```

Comment:
It could be considered a regression.
I'd like to eventually use Open Clover as the source for coveralls, but that is not currently supported.